### ISSUE TITLE
Compiling with -l fails 

### ISSUE THREAD

**Issue by seanmcl** [souffle-lang/souffle#714]
My programs are compiling and running without -l, but giving the following error with -l:

```
$ souffle -p /tmp/foo.prof -l -cw db.dl
failed to compile C++ source <./souffleL2H1Hj.cpp>
```

How can I help debug this?  I am happy to share a file with the team if it helps.

```
$ git log -1
commit a2d4ecef44b4b80c326acc648ec422f00c20a744 (HEAD -> master, origin/master, origin/HEAD)
Merge: 76881a09 269fd0a1
Author: mmcgr <mmcgr@users.noreply.github.com>
Date:   Wed Sep 5 15:58:14 2018 +1000

    Merge pull request #711 from taipan-snake/btree-optimization-new-3

    Improved Provenance via Synthesiser-generated data-structures
```

**Comment by mmcgr** [comment]
There should be a souffle[xxxxxx].[pid].ccerr file generated if compilation fails. That contains the compiler output and should give more information about the error. If you could share that file it would be helpful.

The most likely problem is that it's not finding the headers, but the ccerr file should tell us more.

**Comment by seanmcl** [comment]
Yep

```
$ cat soufflenziSHX.83219.ccerr
./soufflenziSHX.cpp:4:10: fatal error: profile/Tui.h: No such file or directory
 #include "profile/Tui.h"
          ^~~~~~~~~~~~~~~
compilation terminated.
```

What should I do?

**Comment by b-scholz** [comment]
It looks like that the profiler includes are not registered in configure.ac. 
Is this correct Martin?

**Comment by mmcgr** [comment]
It looks like a recent PR had some issues with merge conflicts. That #include should read
`#include "souffle/profile/Tui.h"`

Try checkout out the previous version, commit 76881a09 If you rebuild and reinstall, it should work.

**Comment by mmcgr** [comment]
Fixed by #717
